bootstrap.sh

```diff
@@ -1 +1,21 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
+kubectl create namespace mysql
+
+# Apply MySQL StatefulSet and ConfigMap
+kubectl apply -f manifests/mysql/mysql-secret.yml
+kubectl apply -f manifests/mysql/init-configMap.yml
```

Comment:
The path 'manifests/mysql/init-configMap.yml' should be '.infrastructure/mysql/init-configMap.yml' to match the directory structure provided in your files.